### PR TITLE
Add missing hooks actionCustomerLogout & displayCheckoutSummaryTop

### LINKED ISSUE #19894: No db entry for actionCustomerLogoutBefore or actionCustomerLogoutAfter
#### Describe the bug

There are no entries in the ps_hook table for actionCustomerLogoutBefore or actionCustomerLogoutAfter, so you can't hook modules to them.

I'm testing on 1.7.6, but I don't see them in the database for 1.7.7 either.

Adding the entries to the db, resolves the issue

#### Expected behavior

You should be able to hook to these positions.

#### Steps to Reproduce

1. With a module that provides hook code to these hook, try to transplant the module to them.

**Screenshots**

#### Additional information

* PrestaShop version: 1.7.6
* PHP version: 7.2

### PR DESCRIPTION
**Pull request by ghost**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add missing hooks actionCustomerLogout After Before and displayCheckoutSummaryTop
| Type?             | bug fix
| Category?         | IN
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/PrestaShop/issues/19894 AND https://github.com/PrestaShop/PrestaShop/issues/15594
| How to test?      | Install a new shop and check ps_hook table
| Possible impacts? | Please indicate what parts of the software we need to check to make sure everything is alright.

This PR close this one https://github.com/PrestaShop/PrestaShop/pull/26072 for better commit name

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26072)

### CODE REVIEW

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
I'm sorry, upgrade scripts are going to be removed from the Core, you can do it in the autoupgrade module instead :)
But we can keep the hook.xml file modification of course :)

**Comment by okom3pom** [comment]
I port all the content of the file or only my modification ?

**Comment by ghost** [comment]
@PierreRambaud  https://github.com/PrestaShop/autoupgrade/pull/422

I put the SQL part to the autoupgrade module.

**Review by matthieu-rolland — COMMENTED** [review]
I restarted the UI failing tests, seems like it was a timeout issue

**Review by matthieu-rolland — DISMISSED** [review]

File: install-dev/upgrade/sql/1.7.9.0.sql [hunk 1]
```diff
@@ -11,4 +11,4 @@ INSERT IGNORE INTO `PREFIX_hook` (`id_hook`, `name`, `title`, `description`, `po
   (NULL, 'actionValidateOrderAfter', 'New Order', 'This hook is called after validating an order by core', '1'),
   (NULL, 'actionAdminOrdersTrackingNumberUpdate', 'After setting the tracking number for the order', 'This hook allows you to execute code after the unique tracking number for the order was added', '1'),
   (NULL, 'displayBackOfficeEmployeeMenu', 'Administration menu', 'This hook is displayed in the employee menu', '1')
-;
+;
```
Comment (PierreRambaud) [inline install-dev/upgrade/sql/1.7.9.0.sql]: Don't change this file please :)

Comment (ghost) [reply install-dev/upgrade/sql/1.7.9.0.sql]: Oh my fault it's my IDE I push a new commit for this file will be deleted in any case ?
It's time to try squash ;)  

Sorry

**Pull request closed without merging**